### DISCUSSION TITLE
Make it more clear this is SvelteKit

### DISCUSSION THREAD

**Discussion by MrAmericanMike** [sveltejs/kit#5104] [General]
### Describe the bug

Not much of a "bug" but kinda. As what I'm going to describe, results in things been not so clear and kinda confusing for users.

After using `npm init svelte` you end up with a project in which the README.md contains this among other stuff

```
Everything you need to build a Svelte project, powered by [`create-svelte`](https://github.com/sveltejs/kit/tree/master/packages/create-svelte).
```

But in reality this creates a SvelteKit project, and not just Svelte.

It would be better if running that would either actually create a simple Svelte project, or maybe make it so it's `npm init sveltekit` or any other method that makes it more clear that this is not Svelte, but SvelteKit we are getting into.

`npm init svelte` should be equivalent of a `npx degit sveltejs/template`

I believe that would make things cleaner in away and differentiate between Svelte and SvelteKit

### Reproduction

`npm init svelte`

### System Info

```shell
Not applicable
```

### Severity

annoyance

**Comment by Mlocik97** [comment]
SvelteKit's official description is

> THE FASTEST WAY TO BUILD SVELTE APPS

so yes, it's correct what it says... SvelteKit is framework to build Svelte projects.

It's even planed that template (`sveltejs/template`) will be removed/deprecated when 1.0 will be out, and all Svelte projects will be initialized via `npm init svelte` which will create SvelteKit project.

Btw. that template also isn't just Svelte... Svelte itself is just transpiler that converts Svelte Components to JS code. But it doesn't do anything else, and it's actually rollup plugins for svelte and it's custom server that makes app. This will be in charge of SvelteKit in future.

**Comment by dominikg** [comment]
converting this to a discussion as it kind of is not a bug

**Comment by dominikg** [comment]
create-svelte is part of SvelteKit, so of course it is going to create a sveltekit application.

when you execute it, it greets you with

```
Welcome to SvelteKit!
```
Which should give everyone a good hint of what's happening.

The content of svelte.dev and other relevant documentation on how to start a svelte project are going to be updated to reflect new recommendations. The rollup based sveltejs/template will no longer be recommended, it has not received updates for quite some time.

If you want to start a svelte project without using sveltekit, i recommend you check out `npm init vite` and select one of the svelte templates (svelte or svelte-ts)

**Reply by Mlocik97** [reply]
> The rollup based sveltejs/template will no longer be recommended, it has not received updates for quite some time.

yeah, there is even issue: <https://github.com/sveltejs/template/issues/288>

and I had a long discussion about this on Discord with benmccann  (hope I spelled it correctly this time).

I also would use only Vite's template, if not SvelteKit.

**Reply by MrAmericanMike** [reply]
> > The rollup based sveltejs/template will no longer be recommended, it has not received updates for quite some time.

Thanks, I will have to take a look. My templates are also still using Rollup. I think at some point Snowpack was the recommended too, but looks like that didn't live for too long.

> create-svelte is part of SvelteKit, so of course it is going to create a sveltekit application.

I see that. I guess it would be nice if the script would ask if we want just a Svelte template or a SvelteKit one. Given that it's making use of the name "svelte" to run you would expect that one of the options would be to start a bare Svelte project with it too.

**Comment by dummdidumm** [comment]
I think this is a request to improve the `README.md` that's part of the result of running `npm init svelte`. It's indeed a little confusing that it talks about `create-svelte` when you already done that step / maybe you aren't even aware that `npm init svelte` means "use `create-svelte`" under the hood.
